test/tests/executioners/nl_pingpong/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [./nl_pingpong]
+    type = RunApp
+    input = 'nonlinear_residual_pingpong.i'
+    expect_out = "Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH"
+    requirement = 'MOOSE shall kill execution when non linear residual pinpong is detected.'
```

Comment:
I would reword this as `The system shall consider a nonlinear solve diverged if the nonlinear residual oscillates by a user-controlled number of times.`

Reply:
reworded as suggested